h5glance/terminal.py

```diff
@@ -11,6 +11,21 @@
 from subprocess import run
 import sys
 
+def fmt_dtype(dtype):
+    if dtype.metadata and 'vlen' in dtype.metadata:
+        base_dtype = dtype.metadata['vlen']
+        if base_dtype == str:
+            return 'str'
+        elif base_dtype == 'bytes':
+            return 'bstr'
```

Comment:
What about 'bytes'? ;-)

Reply:
I believe it will be too often confused with [OPAQUE data](http://docs.h5py.org/en/latest/strings.html#how-to-store-raw-binary-data) (which is to me of the most close correspondence to Python3 `bytes`; not HDF5 ASCII string). It is important that HDF5 strings are reported as strings, even if these are ASCII strings.

Reply:
Maybe call them 'UTF-8 string' and 'ASCII string'?

Do we know how OPAQUE data will show up in h5py? I don't think we currently have a case for that.

Reply:
I think OPAQUEs will show up as something containing "void", but I agree that's not much practical interest in such datasets.

'UTF-8 string' and 'ASCII string' seem too much detail to me, given that [H5T_CSET_UTF8 and H5T_CSET_ASCII may well become convertible in the future](http://hdf-forum.184993.n3.nabble.com/Variable-length-string-encoding-conversion-tp4027250p4027256.html). What about 'str' and 'ASCII str' until that happened?

Reply:
That answer is from 2014, and it only says it's possible, not that it's planned. For now, it looks like 'UTF-8 string' and 'ASCII string' are accurate descriptions of the string data types in HDF5. We can always change it later if the HDF5 format changes.